packages/vm-config/README.md

```diff
@@ -0,0 +1,13 @@
+# VM Config
+
+Factored out of @agoric/vats, by way of @agoric/boot.
+
+This is similar to `@agoric/boot` but because that has the integration testing of bootstrap, it depends on almost everything.
+
+The configs themselves have no dependencies. Keeping them separate allows packages to depend on them without depending on the world.
+
+# Future plans
```

Comment:
Plans seem like they belong in issues. Or at least: when we say "we plan to X" I prefer that X is linked to an issue.

And historical info ("Factored out...") belongs in commit messages.

Reply:
not firm enough plans to merit an issue, imo. if it can't be here I would just drop it. 

I could qualify this with "possible future directions"

Reply:
> if it can't be here

It's not a critical thing. I just wonder about it getting out of date. I get the impression that it's likely to be addressed soon anyway.